### ISSUE TITLE
Show different error when same user tries to create a run again

### ISSUE THREAD

**Issue by elpiel-efuture** [assafgelber/slash-coffee#5]
If I call 2 times `/coffee run` it will show me error like:
`elpiel is already on a run! If that's not true, tell them to type `/coffee here` to signal that they came back.`

**Comment by assafgelber** [comment]
Sounds like a great idea and not a very difficult task.
Do you want to take a stab at this yourself?

**Comment by elpiel-efuture** [comment]
Yes I would love to, but I don't have much knowledge in Ruby. But I will try...

**Comment by assafgelber** [comment]
Cool! Feel free to ask any question you may have along the way and I'll help asap.
If you need a pointer, your changes are probably going to be in [this method](https://github.com/assafgelber/slash-coffee/blob/master/app/commands.rb#L21-L39). Good luck!
